### DISCUSSION TITLE
Does it support ElasticSearch?

### DISCUSSION THREAD

**Discussion by abdlkdrgndz** [spatie/laravel-activitylog#1015] [Q&A]
Hello everyone, 

I want to index the data I created here in ElasticSearch. Does ActivityLog support ElasticSearch? I didn't see that in the documentation.

I tried to connect with ElasticSearch but failed.

**(.env) file:**

```
ACTIVITY_LOGGER_DB_CONNECTION=elasticsearch
ACTIVITY_LOGGER_ENABLED=true

ELASTICSEARCH_INDEX=scout
ELASTICSEARCH_HOST=http://localhost:9200
ELASTIC_LOGS_INDEX=logs
SCOUT_DRIVER=elasticsearch
```

**database.php**

```
'connections' => [
        'elasticsearch' => [
            'index'  => env('ELASTICSEARCH_INDEX', 'laravel'),
            'hosts'  => [
                env('ELASTICSEARCH_HOST', 'localhost'),
            ]
        ]
],
```

**Comment by murilolivorato** [comment]
hello  @abdlkdrgndz 
UI wrote this tutorial -

https://medium.com/@murilolivorato/how-to-integrate-elasticsearch-with-laravel-a-step-by-step-guide-e98f0cec7d9a

I do like this , may it can help you
